### DISCUSSION TITLE
how do i live my nextjs code

### DISCUSSION THREAD

**Discussion by swashtika-pms** [community/community#131647] [Programming Help]
### Select Topic Area

Question

### GitHub Feature Area

Actions

### Body

anyone tell

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @swashtika-pms ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion. You are more likely to get a useful response if you are explicit about what your project entails, giving a few more details might help someone give you a nudge in the right direction.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.

**Comment by Rishikesh1601** [comment]
use vercel and upload ur project there, it will get live
